integ-test/src/test/resources/datetime_nested.json

```diff
@@ -0,0 +1,8 @@
+{"index":{"_id":"1"}}
```

Comment:
leverage [YamlRestTest](https://github.com/opensearch-project/sql/blob/main/integ-test/src/yamlRestTest/resources/rest-api-spec/test/issues/3477.yml) in future, avoid unncessary mapping and data files.